packages/react-components/src/components/DevicesButton.tsx

```diff
@@ -285,6 +285,10 @@ export const generateDefaultDeviceMenuProps = (
                 key: camera.id,
                 text: camera.name,
                 title: camera.name,
+                iconProps: {
+                  iconName: 'ContextMenuCameraIcon',
+                  styles: { root: { lineHeight: 0 } }
```

Comment:
I wonder if there is space to make this into its own style somewhere as our icon fix for fluent icons?